source/lib/predicates/number.ts

```diff
@@ -131,4 +131,11 @@ export class NumberPredicate extends Predicate<number> {
 			validator: value => value < 0
 		});
 	}
+
```

Comment:
Add documentation

Reply:
Yes, will fix it as soon as i can!

Reply:
Fixed. I'm wondering if the name should be `integerOrInfinite` though since there is a predicate named `infinite` already?

Reply:
@alexdriaguine Yes, that makes sense.

Reply:
Done